pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -483,10 +483,31 @@ func (pc *PackageControllerClient) Reconcile(ctx context.Context, logger logr.Lo
 // getBundleFromCluster based on the cluster's k8s version.
 func (pc *PackageControllerClient) getBundleFromCluster(ctx context.Context, client client.Client, clusterObj *anywherev1.Cluster) (*releasev1.Image, error) {
 	bundles := &releasev1.Bundles{}
-	nn := types.NamespacedName{
-		Name:      clusterObj.Spec.BundlesRef.Name,
-		Namespace: clusterObj.Spec.BundlesRef.Namespace,
+
+	nn := types.NamespacedName{}
+
+	if clusterObj.Spec.BundlesRef == nil {
```

Comment:
can we leverage the code we have in the cluster package for this? it seems this is duplicated